sos/plugins/zfs.py

```diff
@@ -27,7 +27,7 @@ def setup(self):
             "zpool status -vx"
         ])
 
-        zpools = self.call_ext_prog("zpool list -H -o name")
+        zpools = self.get_cmd_output_now("zpool list -H -o name")
```

Comment:
with `save=False`? (though this might be interesting to have..? (dont know zfs..))